cfme/tests/services/test_provision_stack.py

```diff
@@ -136,27 +149,22 @@ def test_provision_stack(setup_provider, provider, provisioning, dialog, catalog
     Metadata:
         test_flag: provision
     """
-    dialog_name, template_name = dialog
+    dialog_name_new = "dialog_" + fauxfactory.gen_alphanumeric()
+    template_type = provisioning['stack_provisioning']['template_type']
```

Comment:
The orchestration template object seems like it should be presented as a fixture?